### ISSUE TITLE
Wrong handling of elements that contain 'lang' attribute

### ISSUE THREAD

**Issue by dmbogomazov** [AngleSharp/AngleSharp#29]
If input document is encoded in utf8 and contains html elements with 'lang' attribute, value of those elements is mojibake'd
To reproduce this bug I just copied text from MS Word to clipbard and tried to parse resulted html thru AngleSharp library.
Word generates an html with copied text as follows:

```
<html>
<head>
<meta http-equiv=Content-Type content="text/html; charset=utf-8>
</head>
<body lang=RU style='tab-interval:35.4pt'>
<!--StartFragment-->
<p class=MsoNormal><span lang=EN-US style='mso-ansi-language:EN-US'>тест</span><o:p></o:p></p>
<!--EndFragment-->
</body>
</html>
```

To simplify, I removed all extra meta-information about styles and whatever from this sample html.
Text inside span has attribute lang and entire document is already encoded with utf. While traversing node-tree I receive wrong text inside span.

**Comment by FlorianRappl** [comment]
Hi, thanks for the report. I can reproduce the issue, however, placing the `lang` attribute has no effect here and does not play a role (the issue remains when the attributes are removed). This bug will be fixed in v0.7.

**Comment by FlorianRappl** [comment]
v0.7 is out and everything should then also work with the updated NuGet package.
